src/vehicle.h

```diff
@@ -239,7 +239,7 @@ struct vehicle_part : public JsonSerializer, public JsonDeserializer
     int hp() const;
 
     /** parts are considered broken at zero health */
-    bool is_broken() const;
+    bool is_broken() const { return base.damage() >= base.max_damage(); }
```

Comment:
The whole invocation of `damage` here could be dropped by implementing `item.is_broken()` which would compare the internal `damage_` with a value cached in `itype`.